internal/dag/cache.go

```diff
@@ -203,6 +208,46 @@ func (kc *KubernetesCache) Insert(obj interface{}) bool {
 		}
 		kc.httpproxydelegations[m] = obj
 		return true
+	case *serviceapis.GatewayClass:
+		m := toMeta(obj)
+		if kc.gatewayclasses == nil {
+			kc.gatewayclasses = make(map[Meta]*serviceapis.GatewayClass)
+		}
+		// TODO(youngnick): Remove this once service-apis actually have behavior
+		// other than being added to the cache.
+		kc.Infof("Adding GatewayClass %s in namespace %s", m.name, m.namespace)
+		kc.gatewayclasses[m] = obj
+		return true
+	case *serviceapis.Gateway:
+		m := toMeta(obj)
+		if kc.gateways == nil {
+			kc.gateways = make(map[Meta]*serviceapis.Gateway)
+		}
+		// TODO(youngnick): Remove this once service-apis actually have behavior
+		// other than being added to the cache.
+		kc.Infof("Adding Gateway %s in namespace %s", m.name, m.namespace)
```

Comment:
This looks like debugging information, is it needed?

Reply:
If we don't have this in there, there will be no indication whatsoever that the integration is actually working, since there is no coded behavior right now.

Reply:
Who is the consumer of this information? If it’s one of us, debug sounds like a better level. Also, why are the service api types handled specifically, but not the pre exisiting types?

Reply:
I will change to Debug though.